arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/Raise.kt

```diff
@@ -128,7 +165,7 @@ public interface Raise<in R> {
     fold({ raise(it) }, ::identity)
 
   @Deprecated(
-    "Use recover or effect & recover instead",
+    "Use getOrElse on Raise, Effect or EagerEffect instead.",
     ReplaceWith("effect { f() }")
```

Comment:
Should the replacement use `getOrElse` too?

Reply:
This is deprecating the `attempt { f() } catch { g() }` block, but it exists out of two pieces. So I am unsure what is the best thing to do here.

`f` is should be refactored to `Raise`, but that is a different piece in the user code. That is why I recommended `effect { }`. My train of thought:

 1. Refactor `attempt { f() }` to `effect { f() }` user gets rid of the `attempt` call
 2. Refactor `catch` to `getOrElse { g() }` user gets rid of the `catch` call
 3. Refactor `arrow.core.continuations.effect` to `arrow.core.raise.effect`, the lambda body should be source-compatible